lib/model/QubitObject.php

```diff
@@ -305,6 +305,35 @@ public function getStatus($options = [])
         return QubitStatus::getOne($criteria);
     }
 
+    /**
+     * Check to see if note is saved in memory (field was updated).
+     *
+     * Used by ISAD, DACS, and RAD templates when duplicating a description,
+     * specifcally for updating the 'Language and script notes' field.
+     *
+     * @return array $notes
+     */
+    public function getMemoryNotesByType(array $options = [])
+    {
+        // Check memory for notes
+        if (!count($this->notes) < 0) {
+            return $this->getNotesByType($options);
+        }
+
+        $noteTypeId = $options['noteTypeId'];
+        $notes = new ArrayObject();
+
+        foreach ($this->notes as $note) {
+            if ($note->typeId == $noteTypeId) {
+                $notes->append($note);
+            }
+        }
+
+        if (count($notes) > 0) {
+            return $notes;
```

Comment:
It seems like there is a scenario where we end up with 0 `$notes`. Would it be better to skip the if condition and just return `$notes` instead? Right now it'll end up returning `NULL`, which doesn't play with with array methods.